test/test_preproc.py

```diff
@@ -49,6 +60,15 @@ def check_return(result_array, checks):
         ),
         "```fortran90\n#define SUCCESS .true.\n```",
         "```fortran90\nREAL, CONTIGUOUS, POINTER, DIMENSION(:) :: var1\n```",
+        "```fortran90\nINTEGER :: var0\n```",
+        "```fortran90\nREAL :: var1\n```",
+        "```fortran90\nLOGICAL :: var1\n```",
+        "```fortran90\nINTEGER :: var2\n```",
+        "```fortran90\nINTEGER, INTENT(INOUT) :: var\n```",
+        "```fortran90\nREAL(1, 5, 5, 5) :: var3\n```",
+        "```fortran90\nREAL(5, 2, 5, 5) :: var4\n```",
+        "```fortran90\nREAL(5, 5, 5, 4) :: var5\n```",
+        "```fortran90\nREAL(1, 5, 5, 5) :: var6\n```",
```

Comment:
```suggestion
        "```fortran90\nINTEGER, PARAMETER :: res = 0+1+0+0\n```",
        "```fortran90\nINTEGER, PARAMETER :: res = 0+0+0+1\n```",
        "```fortran90\nINTEGER, PARAMETER :: res = 1+0+0+0\n```",
```